src/main/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExt.kt

```diff
@@ -76,3 +78,16 @@ data class EagerResponse(
  * @author sergey
  */
 data class Header(val name: String, val value: String)
+
+internal lateinit var stringMapper: ObjectMapper
```

Comment:
how do u think about property delegation like this:

```suggestion
internal val stringMapper by lazy { ObjectMapper() }
```

Reply:
Good suggestion, I haven't used them before. 👍